test/views/pages/home/content/ranking/static_ranking_widget_test.dart

```diff
@@ -0,0 +1,39 @@
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/views/components/content/publication_header/user_profile_pop_up.dart";
+import "package:proxima/views/pages/home/content/ranking/ranking_widget.dart";
+
+import "../../../../../mocks/providers/provider_homepage.dart";
+
+void main() {
+  testWidgets(
+    "Check that all the ranking elements are displayed correctly",
```

Comment:
Same for the group and naming of tests here.

Reply:
You are right 💯 

Fixed in: [26c5bbb](https://github.com/ProximaEPFL/proxima/pull/322/commits/26c5bbb2d3238231e450bcfae8c13e16a0eb5bec)

Thank you 😄